index.bs

```diff
@@ -65,6 +65,51 @@ Examples {#examples}
     </pre>
 </div>
 
+<div class="example">
+    The following example shows how to use gravity sensor that provides
+    readings in [=screen coordinate system=]. The snippet will print message to the
+    console when the [=dom screen=] is perpendicular to the ground and bottom of the
+    rendered web page is pointing downwards.
+    <pre highlight="js">
+    let sensor = new GravitySensor({frequency: 5, referenceFrame: "screen"});
+
+    sensor.onreading = () => {
+      if (sensor.y >= 9.8) {
+        console.log("Web page is perpendicular to the ground.");
+      } else {
+        console.log("Rotate device.");
+      }
+    }
+
+    sensor.start();
+    </pre>
+</div>
+
+<div class="example">
+    The following example detects shake gesture along x axis of the device, regardless
+    of the orientation of the [=dom screen=].
+    <pre highlight="js">
+    const shakeThreshold = 25;
+    let accel = 0;
+    let lastAccel = 0;
+    let shakeCount = 0;
+
+    let sensor = new LinearAccelerationSensor({frequency: 60});
+
+    sensor.addEventListener('reading', () => {
+      accel = accel * 0.9 + sensor.x - lastAccel;
+      lastAccel = sensor.x;
+
+      if (accel > shakeThreshold) {
```

Comment:
isn't  it simpler just `sensor.x > shakeThreashold`  (similar to https://github.com/intel/generic-sensor-demos/blob/master/punchmeter/main.js#L137) ?

Reply:
Ah, this one has low pass filter and avoids 'recoil' 😄, but for simplicity, sure.

Reply:
done